checkers/mozlenium/mozlenium-test-runner.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+
+/**
+ * Test runner that reads the script provided andtransforms it into something
+ * that will be useful for selenium.
+ */
+class MozleniumTestRunner {
+  static TEST_FILE = './test.js';
```

Comment:
if I'm interpreting this correctly, you're removing the need for https://github.com/mozilla-it/mozalert/blob/main/checkers/mozlenium/entrypoint.sh#L30-L32 which is excellent and this will make local dev a lot easier. To take this a step further it would be great to parameterize 'test.js' so the entrypoint script doesn't need to copy that into place. I could pass in the location of the check and mozlenium can handle it from there. does that seem legit @andrew-sunada ?

Reply:
Yea, for sure! I can make that happen. Thanks for that feedback!